### PR TITLE
bugfix: fix get table meta failed with catalog

### PR DESCRIPTION
**Pull request by l81893521**
### Ⅰ. Describe what this PR did
Can not get table meta when the SQL include catalog, like
```
select * from account.account_tbl
```

### Ⅱ. Does this pull request fix one issue?
#1952

### CODE REVIEW

File: rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/cache/MysqlTableMetaCache.java [hunk 1]
```diff
@@ -102,6 +102,7 @@ protected TableMeta fetchSchema(DataSource dataSource, String tableName) throws
 
     private TableMeta resultSetMetaToSchema(ResultSetMetaData rsmd, DatabaseMetaData dbmd, String tableName)
         throws SQLException {
+        //always "" for mysql
```
Comment (zjinlei) [inline rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/cache/MysqlTableMetaCache.java]: What does this sentence mean?

File: rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/cache/MysqlTableMetaCache.java [hunk 2]
```diff
@@ -122,6 +123,9 @@ private TableMeta resultSetMetaToSchema(ResultSetMetaData rsmd, DatabaseMetaData
             tableName = tableName.replace("`", "");
         }
 
+        String[] catalogTable = tableName.split("\\.");
```
Comment (slievrly) [inline rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/cache/MysqlTableMetaCache.java]: why not use rsmd.getTableName()?

File: rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/cache/MysqlTableMetaCache.java (hunk 1 above)
Comment (l81893521) [reply rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/cache/MysqlTableMetaCache.java]: That means this method always return "" in Mysql. Mysql does not implement this interface.

File: rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/cache/MysqlTableMetaCache.java (hunk 2 above)
Comment (l81893521) [reply rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/cache/MysqlTableMetaCache.java]: Thanks a lot.

**Pull request merged**